doc/release/1.14.1-notes.rst

```diff
@@ -0,0 +1,22 @@
+==========================
+NumPy 1.14.1 Release Notes
+==========================
```

Comment:
@charris: Can you make a separate PR based upon 5f01e54b20e38d483e8bab31bf5f953a860fe8d3 that adds these release notes?

Reply:
Actually - @ahaldane, if you rebase on that commit and split adding/modifying this file in two, that would have the same effect.

Reply:
Did you link the wrong commit? That commit doesn't touch the template part of the release note. Was that the intention?

Reply:
That commit is the original common ancestor between 1.14 and master that I've been basing all my backportable PRs on

Reply:
At risk of being nitpicky, I'd squash your 1st and 3rd commits together, and put them after the template one - that way we can base backportable PRs on top of the template commit

Reply:
`cherry-pick` is so easy, I really don't see the point of using a common base, and If you want to use the same branch, just rebase (onto), force push, and do another PR. If you look at the graph, basing on a point in the past also just fattens up the number of lines from the past.